optimum/intel/openvino/modeling_text2speech.py

```diff
@@ -0,0 +1,678 @@
+                The directory from which to load the model.
+                Can be either:
+                    - The model id of a pretrained model hosted inside a model repo on huggingface.co.
+                    - The path to a directory containing the model weights.
+            save_dir (`str` or `Path`):
+                The directory where the exported ONNX model should be saved, defaults to
+                `transformers.file_utils.default_cache_path`, which is the cache directory for transformers.
+            token (Optional[Union[bool, str]], defaults to `None`):
+                The token to use as HTTP bearer authorization for remote files. If `True`, will use the token generated
+                when running `huggingface-cli login` (stored in `~/.huggingface`).
+            revision (`str`):
+                Revision is the specific model version to use. It can be a branch name, a tag name, or a commit id
+            kwargs (`Dict`, *optional*):
+                kwargs will be passed to the model during initialization
+        """
+        save_dir = TemporaryDirectory()
+        save_dir_path = Path(save_dir.name)
+
+        # This attribute is needed to keep one reference on the temporary directory, since garbage collecting
+        # would end-up removing the directory containing the underlying OpenVINO model
+        cls._model_save_dir_tempdirectory_instance = save_dir
+
+        if task is None:
+            task = cls.export_feature
+            if use_cache:
+                task = task + "-with-past"
+
+        compile_only = kwargs.pop("compile_only", False)
+        if compile_only:
+            logger.warning(
+                "`compile_only` mode will be disabled because it does not support model export."
+                "Please provide openvino model obtained using optimum-cli or saved on disk using `save_pretrained`"
+            )
+            compile_only = False
+        # If load_in_8bit and quantization_config not specified then ov_config is set to None and will be set by default in convert depending on the model size
+        if load_in_8bit is None and not quantization_config:
+            ov_config = None
+        else:
+            ov_config = OVConfig(dtype="fp32")
+        stateful = kwargs.get("stateful", True)
+        variant = kwargs.pop("variant", None)
+
+        # NOTE: need to pass kwargs that can contain vocoder path
+        # it will be an additional component in the pipeline
+        main_export(
+            model_name_or_path=model_id,
+            output=save_dir_path,
+            task=task,
+            subfolder=subfolder,
+            revision=revision,
+            cache_dir=cache_dir,
+            token=token,
+            local_files_only=local_files_only,
+            force_download=force_download,
+            trust_remote_code=trust_remote_code,
+            ov_config=ov_config,
+            stateful=stateful,
+            variant=variant,
+            model_kwargs=kwargs,
```

Comment:
why not add this modification here directly ;
https://github.com/huggingface/optimum-intel/blob/0abe991c2d89ba306cd6078f22f01ebab603cd2a/optimum/intel/openvino/modeling_base_seq2seq.py#L356
(could be useful for more models, if we really want to make sure this is only for text2speech model for now we can still limit it by checking `export_feature`, also avoid code duplication)

Reply:
Done, thanks